templates/lib/prism/serialize.rb.erb

```diff
@@ -82,6 +82,11 @@ module Prism
         source.start_line = load_varsint
       end
 
+      def load_line_offsets
+        offsets = load_varuint.times.map { load_varuint }
+        source.offsets = offsets
```

Comment:
```suggestion
        source.offsets.concat(load_varuint.times.map { load_varuint })
```

Reply:
`concat` makes an extra copy (memcpy/arraycopy of all elements) so that's worse performance-wise.
I'll remove the extra local variable though.

Reply:
In that case can you 

```ruby
load_varuint.times do
  source.offsets << load_varuint
end
```

Reply:
But that will cause extra resizes, same as https://github.com/ruby/prism/pull/2381#discussion_r1488204019